### DISCUSSION TITLE
Limited to 60fps?

### DISCUSSION THREAD

**Discussion by motoleon** [Blinue/Magpie#46] [General]
The game: Enlisted
limit fps in game to 90 fps
vsync off
Monitor is 60hzs
fps option in Magpie to show, and shows 60fps (max).

There is a fps limit in Magpie to 60?

**Comment by Blinue** [comment]
Hello, check out #27

**Reply by motoleon** [reply]
Ok. Thanks.

**Comment by zaifan** [comment]
your monitor is only 60 hz, so fps max is also 60 fps. on my laptop, magpie runs 72 fps because my laptop monitor has 72 hz

**Reply by motoleon** [reply]
Ya, ok. Thanks.
